shuup/importer/importing/importing.py

```diff
@@ -274,6 +273,18 @@ def _handle_special_row_values(self, mapping, value):
 
     def _handle_row_field(self, field, mapping, orig_value, row_session, target, value):
         value = self._get_field_choices_value(field, value)
+
+        # do some magic
```

Comment:
It's magic as long as you don't describe what's happening. Doing magic is in code is bad. Readers should understand what's going on. => Add proper comment, if you think it needs explaining.